cmd/ltctl/main.go

```diff
@@ -138,6 +138,28 @@ func main() {
 	loadtestCmd.AddCommand(loadtestComands...)
 	rootCmd.AddCommand(loadtestCmd)
 
+	sshCmd := &cobra.Command{
+		Use:     "ssh [instance]",
+		Short:   "ssh into instance",
```

Comment:
A user might have trouble finding what name to enter to login to an instance. For example-  "agent-0" or "app-0". I am thinking whether we should clarify the output more prominently in the "deployment info" command, or add another "ssh list" command to list out all the instances.

Because the first question that a user will ask is - how do I get all the list of names to enter ?

Reply:
Right, maybe we can list them in the help context or something like `ssh list` 👍